critiquebrainz/ws/review/views.py

```diff
@@ -225,7 +228,13 @@ def review_modify_handler(review_id, user):
 
     **OAuth scope:** review
 
+    :json string text: Text part of review, min length is 25, max is 5000 **(optional)**
+    :json integer rating: Rating part of review, min is 1, max is 5 **(optional)**
+
+    **NOTE:** The value of unmodified parameter should be set to its value in previous revison.
```

Comment:
Didn't we decide that it's not going to be like that?

Reply:
Yes. I will update it once you confirm these changes https://github.com/metabrainz/critiquebrainz/pull/154/commits/2b1eb3738b80657ffc1fc4dc15c1c7cc16ba8f76